nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,118 @@
+---
+title: SQL Server Transport usage guidance
```

Comment:
@SimonCropp we'll figure it out later. It is [WIP] so not to be merged. It is still debatable if we are going to just include it in the doco content or generate something more like a whitepaper.